internal/util/paramtable/component_param.go

```diff
@@ -83,6 +83,10 @@ func (p *ComponentParam) PulsarEnable() bool {
 	return p.PulsarCfg.Address != ""
 }
 
+func (p *ComponentParam) KafkaEnable() bool {
+	return p.KafkaCfg.Address != ""
```

Comment:
still very uncomfortable about check KafkaEnable
with p.KafkaCfg.Address != ""

Reply:
on the other hand, we also need to consider backward compatibility and keep consistent behavior with other MQ configurations.